addons/website/views/snippets/s_numbers_list.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_list" name="Numbers list">
+    <section class="s_numbers_list o_cc o_cc1 pt80 pb80">
+        <div class="container">
+            <div class="row s_nb_column_fixed">
+                <div class="col-lg-5 pb32">
+                    <h3>Key Metrics of Company's Achievements</h3>
+                    <p><br/>From revenue growth to customer retention and market expansion, our key metrics of company achievements underscore our strategic prowess and dedication to driving sustainable business success.</p>
+                </div>
+                <div class="col-lg-6 offset-lg-1 s_col_no_bgcolor" data-name="Numbers group">
+                    <div class="row">
+                        <div class="col-6 col-lg-6 pt16 pb16">
+                            <h6 class="display-6">15%</h6>
+                            <p>Revenue Growth</p>
+                            <div class="s_hr pt16 pb40">
+                                <hr class="w-100 mx-auto"/>
+                            </div>
+                        </div>
+                        <div class="col-6 col-lg-6 pt16 pb16">
+                            <h6 class="display-6">$50M</h6>
+                            <p>Expected revenue</p>
+                            <div class="s_hr pt16 pb40">
+                                <hr class="w-100 mx-auto"/>
+                            </div>
+                        </div>
+                        <div class="col-6 col-lg-6 pt16 pb16">
+                            <h6 class="display-6">85%</h6>
+                            <p>Customer Retention</p>
```

Comment:
```suggestion
                            <p>User Retention</p>
```
Just for the sake of having it in one line on mobile 🙏🏼
![image](https://github.com/user-attachments/assets/9b77e30e-389e-4183-b1f0-4174cdab8c24)